apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/logs/page.tsx

```diff
@@ -23,71 +44,48 @@ export default async function DocumentPage({
   const { workspace } = await getCurrentUser()
   const projectId = Number(params.projectId)
   const commitUuid = params.commitUuid
+  const documentUuid = params.documentUuid
   const commit = await findCommitCached({ projectId, uuid: commitUuid })
+  const documentLogUuid = searchParams.logUuid?.toString()
+  const page = searchParams.page?.toString?.()
+  const currentLogPage = await fetchDocumentLogPage({
+    workspace,
+    commit,
+    documentLogUuid,
+  })
+
+  if (currentLogPage && currentLogPage !== page) {
+    const route = ROUTES.projects
+      .detail({ id: projectId })
+      .commits.detail({ uuid: commit.uuid })
+      .documents.detail({ uuid: documentUuid }).logs.root
+    return redirect(
+      `${route}?page=${currentLogPage}&logUuid=${documentLogUuid}`,
+    )
```

Comment:
Actually, should we add these parameters in the ROUTES instead? Something like `ROUTES.projects.[…].logs.with({ page, logUuid })`

Reply:
Not today. I have a huge pile of changes to be merged